frontend/packages/dev-console/integration-tests/support/pages/functions/installOperatorOnCluster.ts

```diff
@@ -226,6 +226,11 @@ export const verifyAndInstallPipelinesOperator = () => {
   verifyAndInstallOperator(operators.PipelinesOperator);
 };
 
+export const verifyAndInstallShipwrightOperator = () => {
+  perspective.switchTo(switchPerspective.Administrator);
+  verifyAndInstallOperator(operators.ShipwrightOperator);
+};
+
```

Comment:
It would be great if "shipwright" doesn't exist in the dev-console code and also not in the dev-console tests.

Can we move this function or the two functions in `verifyAndInstallShipwrightOperator` just into `Given('user has installed Shipwright Operator` ... ?

```suggestion
```